test/crash/src/test/java/io/narayana/lra/arquillian/UnmanagedTestBase.java

```diff
@@ -53,7 +55,9 @@ final void stopContainer(String containerQualifier, String deploymentQualifier)
         if (containerController.isStarted(containerQualifier)) {
             LRALogger.logger.debugf("Stopping container %s", containerQualifier);
 
-            deployer.undeploy(deploymentQualifier);
+            if (!deploymentQualifier.equals("")) {
```

Comment:
I'm not fully understanding this, due to my limited knowledge, but will defer to Martin's review of it so I'm not looking for a response.